### PR TITLE
Merge in umee peggo

### PR DESCRIPTION
**Pull request by liamsi**
This repo was based on https://github.com/InjectiveLabs/peggo 
This PR merges in all changes from https://github.com/umee-network/peggo as it contains a bunch of updates and fixes (tooling, deps, tests etc). The smart-contracts are untouched so it should not impact the work @adlerjohn started.

Please **do not squash**

### CODE REVIEW

File: cmd/peggo/client/context.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+import (
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/pkg/errors"
+	umeeapp "github.com/umee-network/umee/app"
```
Comment (liamsi) [inline cmd/peggo/client/context.go:11]: Note to myself: depend on celestia-app here to build client context.

File: cmd/peggo/tx.go [hunk 2]
```diff
@@ -1,178 +1,133 @@
...
 	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
-	"github.com/xlab/closer"
-	log "github.com/xlab/suplog"
-
-	"github.com/InjectiveLabs/peggo/orchestrator/cosmos"
-	"github.com/InjectiveLabs/sdk-go/chain/client"
-	"github.com/InjectiveLabs/sdk-go/chain/peggy/types"
+	"github.com/umee-network/peggo/cmd/peggo/client"
+	"github.com/umee-network/peggo/orchestrator/cosmos"
+	peggytypes "github.com/umee-network/umee/x/peggy/types"
```
Comment (liamsi) [inline cmd/peggo/tx.go:14]: Note to self: I think we need a module in celestia-app for the query types and maybe other bridge related logic. (cc @evan-forbes)

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# NOTE: Order is important; the last matching pattern takes the 
+# most precedence.
+
+# Primary repo maintainers
+*       @alexanderbez
```
Comment (adlerjohn) [inline .github/CODEOWNERS:7]:
```suggestion
* @liamsi
solidity/ @adlerjohn
```

File: .github/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    open-pull-requests-limit: 10
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: daily
+    commit-message:
+      prefix: "chore"
+    open-pull-requests-limit: 10
+    reviewers:
+      - alexanderbez
```
Comment (adlerjohn) [inline .github/dependabot.yml:16]:
```suggestion
      - liamsi
```

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**